library/analytics/build.gradle

```diff
@@ -25,6 +25,9 @@ android {
 dependencies {
     implementation project(':library:base')
 
+    implementation "com.appsee:appsee-android:${deps.appSee.android}"
+    implementation "com.crashlytics.sdk.android:crashlytics:${deps.crashlytics}"
+
```

Comment:
At some point we talked about not including this logic in the analytics module because it would then get included in the instant app

Reply:
Forgot about that so I will move it to the App module

Reply:
you can put the logic in the main app module and still have it listen for analytics events